scripts/build_against_electron.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+source ~/.nvm/nvm.sh
+
+set -e -u
+
+export DISPLAY=":99.0"
+GYP_ARGS="--runtime=electron --target=${ELECTRON_VERSION} --dist-url=https://atom.io/download/electron"
+NPM_BIN_DIR="$(npm bin -g 2>/dev/null)"
+
+function publish() {
+    if [[ ${PUBLISHABLE:-false} == true ]] && [[ ${COMMIT_MESSAGE} =~ "[publish binary]" ]]; then
+        node-pre-gyp package
+        node-pre-gyp publish
+        node-pre-gyp info
+    fi
```

Comment:
You will also need to pass `$GYP_ARGS` to these commands. Otherwise they will default to the wrong runtime/targe.